test/e2e/performanceprofile/functests/utils/cgroup/runtime/runtime.go

```diff
@@ -31,7 +31,7 @@ func GetContainerRuntimeTypeFor(ctx context.Context, c client.Client, pod *corev
 		"/rootfs",
 		"/bin/bash",
 		"-c",
-		fmt.Sprintf("/bin/awk -F '\"'  '/runtime_path.*/ { print $2 }' %s", CRIORuntimeConfigFile),
+		fmt.Sprintf("/bin/ps aux | grep '%s' | grep -oP '(?<=-r\\s)[^\\s]+'", pod.Name),
 	}
```

Comment:
Test looks good to me , May be here we can actually verify if we are getting /usr/bin/crun or /usr/bin/runc and not any other value  ?

Reply:
I think it is more correct to leave this generalized, as we know that the -r conmon argument is container runtime path.
I believe that we should leave the runtime type assertion outside the scope of this function.